src/api/queries/getMetricQueries.js

```diff
@@ -0,0 +1,55 @@
+var timeBound = ['', '1d', '7d', '30d', '60d', '90d', '180d', '365d', '2y', '3y', '5y', '10y']
+var currencies = ['usd']
+
+function optionsChecker (options) {
+  if ('currency' in options) {
+    if (typeof options.currency === 'undefined') {
+      options.currency = 'usd'
+    }
+    if (currencies.indexOf(options.currency) === -1) {
+      throw new Error('This is not a valid currency')
+    }
+  }
+  if ('timeBound' in options) {
+    if (typeof options.timeBound === 'undefined' || options.timeBound === '') {
+      options.timeBound = ''
```

Comment:
```js
if(options.timeBound === '') {
  options.timeBound = ''
}
```
?

Reply:
I think it is a leftover from my last implementation of the whole thing, it is removed now.